wsagent/che-core-api-debug-shared/src/main/java/org/eclipse/che/api/debug/shared/model/action/RunToLocationAction.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2012-2017 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.api.debug.shared.model.action;
+
+/** @author Igor Vinokur */
+public interface RunToLocationAction extends Action {
```

Comment:
RunToLocationAction has to have Location field instead of target and line number.